libs/components/src/next/form/checkbox/stories/vl-checkbox.stories-doc.mdx

```diff
@@ -22,6 +22,21 @@ import { VlCheckboxComponent } from '@domg-wc/components';
 
 <ArgTypes of={VlCheckboxStories.CheckboxDefault} />
 
+### Met value
+
+Indien er geen value meegegeven wordt aan de checkbox, wordt er tijdens het submitten van een form de value `on` teruggegeven.<br/>
+Dit is zoals de HTML5 standaard het voorschrijft. Indien er wel een value meegegeven wordt, zal deze value teruggegeven worden.
+
+<Canvas of={VlCheckboxStories.CheckboxValue} />
+
+## Read only
+
+Aangezien readonly er voor zorgt dat de value van een input niet aangepast kan worden, heeft readonly plaatsen op een checkbox geen effect aangezien het aan- en uitvinken de checked state aanpast en niet de value.<br/>
```

Comment:
grammaticaal niets verkeerd, mss vermijden 2 keer "aangezien" in zelfde zin te gebruiken?

Reply:
Zal het iets beter verwoorden.